.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__/
 /build/
 /dist/
 /tmp/
+spdx/parsers/parsetab.py
```

Comment:
No, this change is not related to Python-3-porting. It just annoyed me that the file was being regenerated all the time on my local Python-2-based environment. Do, would you suggest to upgrade PLY instead? Or is this change still a good one as we should not commit generated files in the first place?